### PR TITLE
Add Dockerfile and instructions to run Win95 as a container

### PR DESCRIPTION
**Pull request by toolboc**
* Added Dockerfile

Confirmed working as-is with Linux, can work elsewhere with X-11 forwarding.  
Docker image currently hosted @ https://hub.docker.com/r/toolboc/windows95/

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+#   If you receive Gtk-WARNING **: cannot open display: unix:0
+#   Run:
+#       xhost +
+#
+
+FROM node:10.9-stretch
+
+LABEL maintainer "Paul DeCarlo <toolboc@gmail.com>"
+
+RUN apt-get update && apt-get install -y \
```
Comment (pojntfx) [inline Dockerfile]: Is there a reason for using `apt-get` instead of the newer and better `apt`?

Comment (toolboc) [reply Dockerfile]: *Updated to the newer better  `apt`

**Comment by jarlah** [comment]
not able to open display on manjaro ;) I could have used more time debugging, but its like, pain and no gain :P
```
[jarlah@jarl-laptop ~]$ sudo docker run -it -v /tmp/.X11-unix:/tmp/.X11-unix -e DISPLAY=unix$DISPLAY --device /dev/snd --name windows95 toolboc/windows95
..........
(electron:73): Gtk-WARNING **: cannot open display: unix:0.0
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! windows95@1.1.0 start: `electron-forge start`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the windows95@1.1.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2018-08-25T16_12_04_598Z-debug.log
[jarlah@jarl-laptop ~]$ echo $XDG_SESSION_TYPE
x11
[jarlah@jarl-laptop ~]$ env | grep -i wayland
[jarlah@jarl-laptop ~]$
```

**Comment by toolboc** [comment]
@jarlah, try running `xhost +` on your system, then try again.

**Comment by toolboc** [comment]
@felixrieseberg , I added a /docs folder with detailed instructions for running and displaying on Linux and Windows (should also work for Mac but untested).  I was able to trim the instructions out of the README and added a link to the new doc per your suggestion.

**Pull request merged**